src/content/blog/2023/hello-nf-core-world.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Hello nf-core world'
+subtitle: 'The inital nf-core blog post'
+headerImage: 'https://images.unsplash.com/photo-1590005354167-6da97870c757'
+headerImageAlt: 'Green apples cut in various ways laying on an bright orange surface'
+pubDate: 2023-11-03T08:00:00.000+01:00
+authors:
+    - 'mashehu'
+    - 'ewels'
+label:
+    - 'community'
+---
+
+## Greetings, nf-core Community
+
+Today marks the launch of the nf-core blog, a new avenue for keeping our community informed and connected.
+This blog will serve as a central hub for in-depth updates and discussions.
+
+## Our Vision for the Blog
+
+This platform is created with the intent to:
+
+-   Share substantive updates from our community’s activities.
```

Comment:
```suggestion
-   Share substantive updates from our community’s activities, like pipeline releases
```

Reply:
or add examples elsewhere. For me it is not super clear from reading this, that it is for pipeline developers to show case their new releases :)